lib/chef/chef_fs/file_system/repository/chef_repository_file_system_root_dir.rb

```diff
@@ -44,6 +44,7 @@
 require "chef/chef_fs/data_handler/user_data_handler"
 require "chef/chef_fs/data_handler/group_data_handler"
 require "chef/chef_fs/data_handler/container_data_handler"
+require "chef"
```

Comment:
and I'm even :-1: on trying to require the resource+provider code directly and bypassing the mess that `require "chef"` will pull in and be more targetted.  we simply haven't architected resources+providers to be that reusable.